pkg/auth/api/proto/auth.proto

```diff
@@ -0,0 +1,1170 @@
+syntax = "proto3";
```

Comment:
there are some calls without auth option... they were used from login. we either have to add an option or remove them from api